lib/rex/ui/text/input/readline.rb

```diff
@@ -17,17 +17,18 @@ class Input::Readline < Rex::Ui::Text::Input
     #
     # Initializes the readline-aware Input instance for text.
     #
-    def initialize(tab_complete_proc = nil)
+    def initialize(tab_complete_proc)
```

Comment:
Looks like this is a backwards breaking change? Was there an issue you were hitting that needed this to be removed? 👀

Reply:
No issue, more that it's only use I could find does pass in a `tab_complete_proc`, and Ideally we want a process to complete tabs. If there's a reason we want to keep the above I'm happy to undo

Reply:
I think holistically the API is sound enough that we can provide Input::Readline with the ability not to provide tab completion 🤷

Reply:
```suggestion
    def initialize(tab_complete_proc = nil)
```

Reply:
👍